docs/getting-started/part-3/metadata.md

```diff
@@ -0,0 +1,87 @@
+
+## About Metadata 
+<!--Should we also write about the markdown description files?-->
+Descriptive metadata, in the context of SDLB, refers to additional configuration attributes and markdown files that goes beyond the basic "id" of DataObjects and Actions. It is mainly about adding descriptions and tags to configuration object, and other attributes that help manage and explore data effectively. 
+
+## Metadata and Descriptions
+<!--There is also a metadata.name, where one can define a user friendly name for the configuration object. -->
+
+While metadata configuration attributes are optional, they can be very useful in many situations. `metadata.name` can define a user frinedly and name stringforward for the configuration object. 
+
+In the realm of SDLB, metadata serves as a valuable space for documenting the usage of custom transformations, providing clarity on the intricacies of the data pipeline.
+For instance, metadata for action `join-departures-airports` can be:
+
+```
+metadata {
+          name = "Airport Departures Join"
+          description = "merging flight details and airport locations"
+          tags = ["merge", "airports", "coordinates"]
+          feed = compute
+    }
+```
+
+SDLB incorporates a robust metadata feature within its configuration file, enhancing the definition of SDL objects with detailed descriptions, labels for layers, subject areas, and tags. This goes beyond mere documentation, offering a comprehensive approach to understanding and managing SDL Objects effectively.
+
+For example, editing metadata for the `ext-departures` data object can be achieved with the following configuration:
+
+```
+  ext-departures {
+    type = WebserviceFileDataObject
+    url = "https://opensky-network.org/api/flights/departure?airport=LSZB&begin=1696854853&end=1697027653"
+    readTimeoutMs=200000
+    metadata = {
+          name = "Flight Departures Web Download"
+          description = "OpenSky Network flight departures"
+          layer = "extern"
+          subjectArea = "flight data"
+          tags = ["aviation", "flight", "departures"]
+    }
+  }
+```
+:::tip UI Tool
+If we check [UI Demo](https://ui-demo.smartdatalake.ch/#/config/dataObjects/ext-departures), the given information in metadata are visualized for `ext-departures`.
+
+![ext-deparures-config](ext-deparures-config.png)
+
+For a deeper understanding of our visualization tool, please refer to this [blog entry](../../../blog/sdl-uidemo).
+:::
+
+## Metadata and Feeds
+<!--On the other side i would reduce the section on feeds, and just refer to the getting started chapter https://smartdatalake.ch/docs/getting-started/part-1/select-columns/#more-on-feeds -->
```

Comment:
Agree, I would remove this section and just mention that the feed we already had configured, is displayed in the UI too.